DeepLinkKit/Router/DPLDeepLinkRouter.m

```diff
@@ -137,6 +137,19 @@ - (BOOL)handleURL:(NSURL *)url withCompletion:(DPLRouteCompletionBlock)completio
 }
 
 
+- (nullable DPLDeepLink *)deepLinkForURL:(nonnull NSURL *)url {
```

Comment:
Sure thing. I’d initially tried to do this, but ran in to some trouble because the when the above method loops through its `routes`, does two things:

1. Checks to see if the route matches the URL.
2. If it finds a route that matches, it handles that `route` in the body of the loop.

So, I wasn’t quite sure how to handle both those things in the same method. I am definitely open to suggestion though!